lms/templates/courseware/courseware.html

```diff
@@ -14,6 +15,11 @@
 </%def>
 
 <%block name="bodyclass">view-in-course view-courseware courseware ${course.css_class or ''}</%block>
+
+<%block name="js_overrides">
```

Comment:
@muhammad-ammar - @tobz points out that it would be more useful if this block was in main.html so that any LMS page is able to load dependencies with the correct hashes. Do you see any reason why that wouldn't work?